packages/astro/src/@types/astro.ts

```diff
@@ -536,6 +536,41 @@ export interface AstroUserConfig {
 	 */
 	redirects?: Record<string, RedirectConfig>;
 
+	/**
+	 * @docs
+	 * @name prefetch
+	 * @type {boolean | PrefetchOptions}
+	 * @description
+	 * Enable prefetching for your site. A prefetch script is automatically added to every page
+	 * in the project. Just add the `data-astro-prefetch` attribute to any `<a />` links on your page
+	 * and you're ready to go!
+	 *
+	 * If you use [View Transitions](https://docs.astro.build/en/guides/view-transitions/), prefetch will
+	 * be enabled automatically with a default of `{ prefetchAll: true }`, which enables prefetching for all links.
+	 * You can configure this option to override the defaults, and even set `false` to disable prefetching
+	 * completely when using View Transitions.
+	 */
+
+	/**
+	 * @docs
+	 * @name prefetch.prefetchAll
+	 * @type {boolean}
+	 * @description
+	 * Enable prefetching for all links by default, similar to all `<a />` links having the `data-astro-prefetch`
+	 * attribute set. You can disable prefetching individually by setting `data-astro-prefetch="false"` on a link.
+	 */
+
+	/**
+	 * @docs
+	 * @name prefetch.defaultStrategy
+	 * @type {'tap' | 'hover' | 'viewport'}
+	 * @default `'hover'`
+	 * @description
+	 * The default prefetch strategy to use when the `data-astro-prefetch` attribute (no value) is set on a link.
+	 * You can select a different strategy for individual links by setting, e.g. `data-astro-prefetch="viewport"`.
```

Comment:
```suggestion
	 * The default prefetch strategy to use when the `data-astro-prefetch` attribute is set on a link with no value.
	 * 
	 * - `'tap'`: Prefetch just before you click on the link.
	 * - `'hover'`: Prefetch when you hover over or focus on the link. (default)
	 * - `'viewport'`: Prefetch as the links enter the viewport.
	 * 
	 * You can override this default value and select a different strategy for any individual link by setting a value on the attribute.
	 *
	 * ```html
	 *  `<a href="/about" data-astro-prefetch="viewport">About</a>`
	 * ```
```
I know this is all in the guide, but when there are options, I do think it makes sense to have them in reference for quick "lookupability"!